resources/splunk_install.rb

```diff
@@ -0,0 +1,223 @@
+# Encoding: utf-8
+#
+# Cookbook Name:: cerner_splunk_ingredient
+# Resource:: splunk_install
+#
+# Resource for managing the installation of Splunk
+class SplunkInstall < ChefCompat::Resource
+  include CernerSplunk::PlatformHelpers, CernerSplunk::PathHelpers
+
+  resource_name :splunk_install
+
+  property :name, String, name_property: true, identity: true
+  property :package, [:splunk, :universal_forwarder], required: true
+  property :version, String, required: true
+  property :build, String, required: true
+  property :base_url, String, default: 'https://download.splunk.com/products'
+  property :accept_license, [TrueClass], required: true
```

Comment:
That seems dissonant though. In the README, we outline that true or false are valid options for this attribute, but here we fail with a validation error if false is actually used.